app/assets/javascripts/date_range_picker.js.erb

```diff
@@ -2,7 +2,13 @@
 // All this logic will automatically be available in application.js.
 
 $(document).ready(function() {
-  var rails_right_now = '<%= Time.zone.now.strftime('%Y%m%d') %>';
```

Comment:
Ahh, I see the problem here. This `Time.zone.now` probably only gets evaluated every time we deploy and gets stale! Good catch!